packages/kit/src/core/sync/sync.js

```diff
@@ -57,13 +58,14 @@ export async function update(config, manifest_data, file) {
  */
 export async function all(config, mode) {
 	init(config, mode);
-	return await create(config);
+	return await create(config, mode);
 }
 
 /**
  * Regenerate server-internal.js in response to src/{app.html,error.html,service-worker.js} changing
  * @param {import('types').ValidatedConfig} config
+ * @param {string} mode
  */
-export function server(config) {
-	write_server(config, path.join(config.kit.outDir, 'generated'));
+export function server(config, mode) {
+	write_server(config, path.join(config.kit.outDir, 'generated'), mode);
```

Comment:
Does anyone know where this function is used? I couldn't find any implementations within the repo to update to match the new signature. Thanks!

Reply:
it's used in `vite/dev/index.js` — for future reference, if you're using VSCode you can find things like this with the 'Find All References' menu option

<img width="954" alt="image" src="https://user-images.githubusercontent.com/1162160/213577527-7049367b-231e-41e1-a333-4cb8eaf7f654.png">

Reply:
Ah good one, thank you!